deploy/infra/groups/data.bicep

```diff
@@ -46,12 +46,40 @@ param synapseMIStorageAccountRoles array = [
 ]
 param synapseMIPrincipalId string = ''
 
+// Name parameters for Postgres
+param serverName string = ''
+param administratorLogin string = ''
+param postgresAdminLoginPass string = '' 
+
+// Parameters with default values for Postgres
+param skuCapacity int = 2
+param skuName string = 'GP_Gen5_2'
+param skuSizeMB int = 51200
+param skuTier string = 'GeneralPurpose'
+param skuFamily string = 'Gen5'
+param postgresqlVersion string = '11'
+param backupRetentionDays int = 7
+param geoRedundantBackup string = 'Disabled'
+
+param postgresAdminPasswordSecretName string = 'PostgresAdminPassword'
+param utcValue string = utcNow()
+
+var postgresAdminLoginPassVar = empty(postgresAdminLoginPass) ? '${uniqueString(resourceGroup().id)}!' : postgresAdminLoginPass
+var administratorLoginVar = empty(administratorLogin) ? '${environmentCode}_admin_user' : administratorLogin
+var serverNameVar = empty(serverName) ? '${environmentCode}-pg-server' : serverName
+
+param uamiName string = ''
+var uamiNameVar = empty(uamiName) ? '${namingPrefix}-umi' : uamiName
+
 var namingPrefix = '${environmentCode}-${projectName}'
 var dataResourceGroupNameVar = empty(dataResourceGroupName) ? '${namingPrefix}-rg' : dataResourceGroupName
 var nameSuffix = substring(uniqueString(dataResourceGroupNameVar), 0, 6)
 var keyvaultNameVar = empty(keyvaultName) ? '${namingPrefix}-kv' : keyvaultName
 var rawDataStorageAccountNameVar = empty(rawDataStorageAccountName) ? 'rawdata${nameSuffix}' : rawDataStorageAccountName
 
+param firewallAllowEndIP string = '255.255.255.255'
```

Comment:
we should restrict this by default. Can we make this default allowing only Azure services that way once deployed pipeline should work as is. Lets leave a note in the README about adding required ip access in the firewall.